idd/Energy+.idd.in

```diff
@@ -38873,34 +38870,22 @@ AirTerminal:SingleDuct:VAV:Reheat:VariableSpeedFan,
        \minimum 0.0
        \maximum 1.0
        \note fraction of cooling air flow rate
-  A3,  \field Air Inlet Node Name
```

Comment:
So, these node names were marked as deprecated way back in 2008 as part of CR7533/#2316  
@kbenne does OpenStudio write anything in the AirTerminal:SinglDuc:VAV:Reheat:VariableSpeedFan (UFAD) fields
A3 Air Inlet Node Name
A4 Air Outlet Node Name
Would it be a problem if these nodes went away along with other deprecated fields?  The reason I ask about these two in particular is because this leaves AirTerminal:SinglDuc:VAV:Reheat:VariableSpeedFan without any node names in it.